app/controllers/api/v1/users_controller.rb

```diff
@@ -6,8 +6,28 @@ class UsersController < ApplicationController
       attr_accessor :user
 
       def index
```

Comment:
Please refactor this action logic, it's seems not good variant of implementation. Think about that